library.properties

```diff
@@ -1,5 +1,5 @@
 name=NimBLE-Arduino
-version=1.3.6
+version=1.3.9
```

Comment:
Shouldn't this be 1.3.8?

Reply:
As 1.3.8 is already released, I would think this is set for the next release

Reply:
Next release will probably be 2.0.0, I think we should have the master set to the same version number as the release/1.3  branch for now to avoid confusion.